Docker/Jenkins2/jenkins-master-deployment.yaml

```diff
@@ -0,0 +1,355 @@
+{{- if .Capabilities.APIVersions.Has "apps/v1" }}
```

Comment:
This doesn't look like official Jenkins [chart](https://github.com/helm/charts/blob/master/stable/jenkins/templates/jenkins-master-deployment.yaml). Is it some simplified version?

Reply:
I've forked the original one and made some changes.
This is not one of them, perhaps the chart / tarball I've downloaded with `helm fetch` was from an older version, hence, the mismatch with the one from your link.